examples/gno.land/r/demo/foo20-airdrop/airdrop_test.gno

```diff
@@ -0,0 +1,65 @@
+package foo20airdrop
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/airdrop"
+	"gno.land/p/demo/merkle"
+	"gno.land/r/demo/foo20"
+	"gno.land/r/demo/users"
+)
+
+var leaves []merkle.Hashable = []airdrop.AirdropData{
```

Comment:
What do you think about moving this to `TestClaimAirdrop`? (see my next comment about removing `TestRegisterMerkle`)